game-core/src/main/java/games/strategy/triplea/pbem/AxisAndAlliesForumPoster.java

```diff
@@ -250,23 +189,12 @@ public IForumPoster doClone() {
     return clone;
   }
 
-  @Override
-  public boolean supportsSaveGame() {
-    return true;
-  }
-
-  @Override
-  public void viewPosted() {
-    OpenFileUtility.openUrl(UrlConstants.AXIS_AND_ALLIES_FORUM + "?topic=" + m_topicId + ".10000");
-  }
-
-  @Override
-  public String getTestMessage() {
-    return "Testing, this will take about 20 seconds...";
-  }
-
   @Override
   public String getHelpText() {
     return HelpSupport.loadHelp("axisAndAlliesForum.html");
```

Comment:
Forgot to mention this in the last PR, but you probably want to adjust this file to match the new code.

Reply:
Good catch. Updated it.